armtemplates/alerts/azuredeploy_Disk.parameters.json

```diff
@@ -12,16 +12,16 @@
             "value": "YourActionGroupName"
         },
         "alertName": {
-            "value": "CPU Alert"
+            "value": "Disk Alert"
         },
         "searchName": {
-            "value": "CPUPercentTest"
+            "value": "DiskPercentTest"
         },
         "scheduleName": {
             "value": "Schedule"
         },
         "customEmailSubject": {
-            "value": "CPU alert"
+            "value": "Disk alert"
```

Comment:
If I were to receive this email, I might prefer something like "ALERT: Disk space low!" which would both catch my attention and make it easy to search for all of the "ALERT" emails